### PR TITLE
Only display the visionneuse button if pertinent

### PR DESCRIPTION
**Pull request by davidbgk**

### CODE REVIEW

File: repondeur/zam_repondeur/views/lectures.py [hunk 1]
```diff
@@ -87,27 +87,27 @@ def __init__(self, request: Request) -> None:
...
-            DBSession.query(Amendement)
-            .filter(
-                Amendement.chambre == self.lecture.chambre,
-                Amendement.session == self.lecture.session,
-                Amendement.num_texte == self.lecture.num_texte,
-            )
-            .count()
+        amendements_count = self.amendements.count()
+        can_be_displayed = any(
+            [amd.avis or amd.gouvernemental for amd in self.amendements.all()]
```
Comment (ronnix) [inline repondeur/zam_repondeur/views/lectures.py]: Ça marche sans les crochets ?

File: repondeur/zam_repondeur/views/lectures.py [hunk 2]
```diff
@@ -87,27 +87,27 @@ def __init__(self, request: Request) -> None:
         )
         if self.lecture is None:
             raise HTTPNotFound
+        self.amendements = DBSession.query(Amendement).filter(
```
Comment (ronnix) [inline repondeur/zam_repondeur/views/lectures.py]: Rename to `amendements_query` for clarity ? I thought it was a list of results at first.

File: repondeur/zam_repondeur/views/lectures.py (hunk 1 above)
Comment (ronnix) [reply repondeur/zam_repondeur/views/lectures.py]: Ça marche sans le .all() ?

Comment (davidbgk) [reply repondeur/zam_repondeur/views/lectures.py]: 👍

File: repondeur/zam_repondeur/views/lectures.py (hunk 2 above)
Comment (davidbgk) [reply repondeur/zam_repondeur/views/lectures.py]: 👌

**Pull request merged**